### PR TITLE
feat: text-utilities, lists

### PR DESCRIPTION
**Pull request by ddivyansh18**
- Various utilities to decorate text
- Normal list and stacked (notification) list

### CODE REVIEW

**Review by romabulani — APPROVED** [review]
Looks good to merge

**Review by manish591 — APPROVED** [review]
Looking good to merge.

File: Components/Text-Utilities/text.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.small-text {
+  font-size: 0.6rem;
+}
+
+.gray-text {
+  color: gray;
```
Comment (rohit8686) [inline Components/Text-Utilities/text.css:6]: Everything looks good but you can give variable for color

**Review by rohit8686 — APPROVED** [review]

File: Components/Text-Utilities/text.css (hunk 1 above)
Comment (ddivyansh18) [reply Components/Text-Utilities/text.css:6]: Thanks for the suggestion, I have variables defined for colours in default.css, but given that "gray" is a color having a predefined proper name in CSS, I think due to that no reason to use a CSS variable in this case.

**Pull request merged**